### ISSUE TITLE
Motion Vector AOV

### ISSUE THREAD

**Issue by thiagodesul** [LuxCoreRender/LuxCore#284]
Hello! Are you planning to add support to Motion Vector AOV compatible with Nuke?

**Comment by Dade916** [comment]
Are you looking for "motion blur" motion vector or "animation" motion vector ?

I ask because LuxCore has motion blur information so it is technically possible to have a Motion Vector AOV (only of motion blurred objects). 
However LuxCore doesn't usually have animation information so anything related is not possible.

**Comment by thiagodesul** [comment]
> However LuxCore doesn't usually have animation information so anything related is not possible.

Could you clarify it better? At the moment Lux is not compatible with animations?

**Comment by thiagodesul** [comment]
> Are you looking for "motion blur" motion vector or "animation" motion vector ?

I'm looking for something like this

https://docs.redshift3d.com/display/RSDOCS/Integrated+AOVs#IntegratedAOVs-MotionVectors

**Comment by Dade916** [comment]
> > However LuxCore doesn't usually have animation information so anything related is not possible.
> 
> Could you clarify it better? At the moment Lux is not compatible with animations?

Lux renders a single image and it is result of a static scene. Animations are the result of rendering multiple scenes. Multiple scene can be result of multiple full scene export from Blender (like in current BlendLuxCore support for animation) or incremental editing of a scene (like in current BlendLuxCore support for viewport rendering).

As far as I know, this is the behavior of any render engine available for Blender, including Cycles.

**Comment by Dade916** [comment]
> > Are you looking for "motion blur" motion vector or "animation" motion vector ?
> 
> I'm looking for something like this
> 
> https://docs.redshift3d.com/display/RSDOCS/Integrated+AOVs#IntegratedAOVs-MotionVectors

Yes, they seem to talking of "motion blur" motion vector.

**Comment by juangea** [comment]
Yes, these is motion vectors for motion blur, I'm not even sure in Cycles there is a motion pass like the one you described.

The Motion Vectors pass is kind of important because it helps us to accelerate render while having the ability to have motion blur in post :)

**Comment by thiagodesul** [comment]
No in Cycles there wasn't proper motion vector AOV like this. I think Lux has a lot of potential, features like this would help Lux to be used in production.

**Comment by thiagodesul** [comment]
And this for sure would help to bring more users and donors on board! You're doing such a great job! I need to say that Lux doesn't stop to amaze me!
